tardis/simulation/base.py

```diff
@@ -109,6 +109,7 @@ class Simulation(PlasmaStateStorerMixin, HDFWriterMixin):
         "iterations_t_rad",
         "iterations_electron_densities",
         "iterations_t_inner",
+        "config"
```

Comment:
The config is not yet a supported hdf property.

Reply:
yep - need to rebase